Src/AmrCore/AMReX_ErrorList.H

```diff
@@ -378,7 +378,7 @@ std::ostream& operator << (std::ostream& os, const ErrorList& elst);
 
   struct AMRErrorTagInfo
   {
-    int m_max_level = 100000;
+    int m_max_level = 1000;
```

Comment:
Note: I changed this just because we're now storing a vector of length m_max_level and I wanted to avoid it being gigantic, and no one will realistically have more than 1000 levels.